### PR TITLE
Atoms spinner

### PR DESCRIPTION
**Pull request by mboperator**
## Changes
- Ported Spinner styles from `Particles`
- Made conform to BEM

## Screenshot
screen shot 2017-09-27 at 12 11 13 pm

### CODE REVIEW

File: src/styles/components/_spinner.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.spinner {
+  box-sizing: border-box;
+  display: block;
+  margin: 0;
+  min-height: $spinner-container-min-height;
+  padding: 0;
+  position: relative;
+
+  &::before {
+    content: attr(data-text);
```
Comment (brock-noah) [inline src/styles/components/_spinner.scss]: I'd be down to rename this to something hinting at spinner, like `data-spinner-text`. Even though current procore uses data-text, I think it is a little vague.

File: src/styles/components/_spinner.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
+@import '../settings/variables';
+
+.spinner {
```
Comment (brock-noah) [inline src/styles/components/_spinner.scss]: Every CSS selector is global. Anything to reduce collisions I think is a good idea. I think we should namespace atoms with 2 or 3 letters bit, `pc-spinner`, `pro-spinner`, `pct-spinner`.

Comment (mboperator) [reply src/styles/components/_spinner.scss]: Thanks for the comment!

Webpack actually prepends every class with `atom-` :)

File: src/styles/components/_spinner.scss (hunk 1 above)
Comment (mboperator) [reply src/styles/components/_spinner.scss]: How about `data-spinner-label`?

**Pull request merged**